src/Adapter/MSTest.TestAdapter/Execution/TestExecutionManager.cs

```diff
@@ -438,8 +438,7 @@ private void ExecuteTestsWithTestRunner(
             PlatformServiceProvider.Instance.AdapterTraceLogger.LogInfo("Executing test {0}", unitTestElement.TestMethod.Name);
 
             // Run single test passing test context properties to it.
-            IDictionary<TestProperty, object?> tcmProperties = TcmTestPropertiesProvider.GetTcmProperties(currentTest);
```

Comment:
This dictionary was always empty because we don't set `TestCaseIdProperty` anywhere, so GetTcmProperties returns an empty dictionary right away.